### ISSUE TITLE
Lower performances than Detectron2

### ISSUE THREAD

**Issue by hgjung3** [open-mmlab/mmdetection#4019]
For MS COCO, mmdetection produces **39.8 AP** that is lower than **42.0 AP** reported in https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md#faster-r-cnn

It seems that the two open toolboxes use slightly different hyper-parameter settings but it is quite difficult for me to compare them one-by-one.
Are there some suggestions to improve mmdetection with R101 on COCO **using basic tricks** such as additional data augmentation?

Thanks in advance!

**Comment by shinya7y** [comment]
In addition to the difference of learning rate schedule, multi-scale training is important.
Please refer [this config](https://github.com/open-mmlab/mmdetection/blob/master/configs/faster_rcnn/faster_rcnn_r50_caffe_fpn_mstrain_1x_coco.py) to reproduce `MIN_SIZE_TRAIN: (640, 672, 704, 736, 768, 800)` in [Detectron2](https://github.com/facebookresearch/detectron2/blob/master/configs/Base-RCNN-FPN.yaml).

**Comment by hgjung3** [comment]
Thanks for the comment! I now see the difference, and in my custom dataset the multi-scale training actually affects the performances of mmdetection.
